Lib/test/test_http_cookiejar.py

```diff
@@ -174,12 +174,10 @@ def test_iso2time_garbage(self):
             '1980-01-01 00:61:00',
             '01-01-1980 00:00:62',
             '01-01-1980T00:00:62',
-            '19800101T250000Z'
-            '1980-01-01 00:00:00 -2500',
+            '19800101T250000Z',
```

Comment:
This looks suspicious to me. Why is the line starting with `1980-01-01` dropped?

Reply:
Because it is in valid format. Test fails because `iso2time()` returns not None. I don't know what invalid example was meant, so I have deleted it.

Reply:
Ah. Okay.